src/components/ShowList.jsx

```diff
@@ -22,6 +21,30 @@ function ShowList() {
     setFilter(e.target.value);
   };
 
+  const inactiveItem = (item) => {
+    const now = new Date().getTime() / 1000;
+    let elapsedTime =
+      item.lastPurchasedDate != null
+        ? (now - item.lastPurchasedDate.seconds) / (60 * 60 * 24)
+        : item.nextPurchase;
+    if (item.numberOfPurchases < 2 || item.nextPurchase * 2 <= elapsedTime) {
+      return true;
+    }
+    return false;
+  };
+
+  const sortItems = (doc1, doc2) => {
```

Comment:
For readability, I'd use `item1, item2` or something as long as it has `item` in the name. `doc` is a general term used by firebase, or all document databases, to represent a concept.

Reply:
I would second Kevin here. The naming can make it more descriptive.